pkg/docker/cockpit-atomic-storage

```diff
@@ -420,16 +420,12 @@ def cmd_monitor():
     mon = subprocess.Popen([ "stdbuf", "-o", "L", "udevadm", "monitor", "-u", "-s", "block"],
                            bufsize=1, stdout=subprocess.PIPE)
     while mon.poll() == None:
-        (r, _, _) = select.select([ mon.stdout ], [] , [], 5)
-        got_event = False
-        if len(r) > 0:
-            if mon.stdout.readline().decode().startswith("UDEV  "):
-                got_event = True
-        # XXX - skip udev/lvm stuff on pure timeouts
-        info = get_info()
-        if info != old_info:
-            print_info(info)
-            old_info = info
+        if select.select([mon.stdout], [] , [], 5)[0] and mon.stdout.readline().decode().startswith("UDEV  "):
+            # XXX - skip udev/lvm stuff on pure timeouts
+            info = get_info()
+            if info != old_info:
+                print_info(info)
+                old_info = info
```

Comment:
We should also update the info once every 5 seconds, not only on udev events.  The usage numbers in the info need to be polled.

So we need to run get_info every time `select` returns, but we should only read a line if `select` says that there is stuff to read.

To get rid of the warning, just remove the assignments to `got_event`.  It was meant to be used for deciding whether to do the "udev/lvm stuff" in addition to the usage calculation, but I never bothered to make that optimization...

Or we could make `got_event` a (unused) parameter to `get_info`.